corehq/apps/smsbillables/models.py

```diff
@@ -367,3 +399,51 @@ def _get_usage_fee(cls, domain, direction):
                 % (direction, domain)
             )
         return usage_fee
+
+    @classmethod
+    def _get_twilio_charges(cls, backend_message_id, direction, couch_id):
+        def _get_twilio_client():
+            twilio_backend = SQLTwilioBackend.get_global_backends_for_this_class(SQLTwilioBackend.SMS).get()
```

Comment:
i think it's also just more performant to do the targeted lookup since we index on the backend id and also the method i referenced does quickcaching, so i'd be in favor of updating it